.github/build-native-images.sh

```diff
@@ -17,5 +17,31 @@ do
         ${BUILD_ENGINE} --tag="${PREFIX_NAME}:${version}"
 done
 
+# Create floating tags
+# For example 19.3-java8, 20.0-java8, 20.0-java11
+# The tag target the latest minors. So if there are versions 19.3.2 and 19.3.3, it will create a tag pointing to 19.3.3 named 19.3-java[8|11]
+
+# Get all the versions, reverse sort them, and keep only the first for each major.minor (e.g. 20.1)
+LATEST_JAVA8_VERSIONS=$(tr ' ' '\n'  <<< "${VERSIONS[@]}" | grep java8 | sort -r | sort -k1,1 -k2,2 -k5,5 -t'.' --unique)
```

Comment:
If you use `-k4,4` here you won't need to have two variables (one for java8 and one for java11), and it will also be future proof (i.e. support java17)
```suggestion
LATEST_JAVA_VERSIONS=$(tr ' ' '\n'  <<< "${VERSIONS[@]}" | sort -r | sort -k1,1 -k2,2 -k4,4 -t'.' --unique)
```
Then you can use `java_version` in `tag` as in my previous comment

Reply:
That does not work for me. I only get java8 version

Reply:
You are right, and my test is wrong. Which makes clear why all this hackery is not a good idea :)

Reply:
Yeah... I guess we will go with this for the time being, and start rewriting it with jbang next week.